dawn/renderer/actions/test/EditorActions.test.js

```diff
@@ -0,0 +1,58 @@
+import { expect } from 'chai';
+import { openFile, saveFile, deleteFile, createNewFile, editorUpdate, changeTheme, increaseFontsize, decreaseFontsize } from '../EditorActions';
+
+
+describe('editor actions creator', () => {
+  // Needs to be using the same seed as the action creator.
+  it('should create an action to open file', () => {
+    const expectedAction = {
+      type: 'OPEN_FILE',
+    };
+    expect(openFile()).to.deep.equal(expectedAction);
+  });
+  it('should create an action to save file', () => {
+    const expectedAction = {
+      type: 'SAVE_FILE',
+      saveAs: false
+    };
+    expect(saveFile()).to.deep.equal(expectedAction);
+  });
+  it('should create an action to delete file', () => {
+    const expectedAction = {
+      type: 'DELETE_FILE',
+    };
+    expect(deleteFile()).to.deep.equal(expectedAction);
+  });
+  it('should create an action to create new file', () => {
+    const expectedAction = {
+      type: 'CREATE_NEW_FILE',
+    };
+    expect(createNewFile()).to.deep.equal(expectedAction);
+  });
+  it('should create an action to update editor', () => {
+    const expectedAction = {
+      type: 'UPDATE_EDITOR',
+      code: 1337,
```

Comment:
in this case, `code` should be a string and not a number. It still works either way, but I think it'll be more realistic to use a string here.